frontend/common/src/components/UserProfile/ProfileSections/DiversityEquityInclusionSection.tsx

```diff
@@ -68,35 +100,153 @@ const DiversityEquityInclusionSection: React.FunctionComponent<{
                   "Label preceding what groups the user identifies as part of, followed by a colon",
               })}{" "}
             </p>{" "}
-            <ul
-              data-h2-font-weight="base(700)"
-              data-h2-padding="base(0, 0, 0, x1)"
+            {(isIndigenous ||
+              (indigenousCommunities && indigenousCommunities.length > 0)) && (
+              <div
+                data-h2-padding="base(x1, 0, 0, 0)"
+                data-h2-display="p-tablet(flex)"
+                data-h2-align-items="base:(center)"
+                data-h2-justify-content="base:(space-between)"
+              >
+                <ul data-h2-padding="base(0, 0, 0, x1)">
+                  <li>
+                    {intl.formatMessage(
+                      getEmploymentEquityStatement("indigenous"),
+                    )}
+                  </li>
+                  <ul data-h2-padding="base(0, 0, 0, x1)">
+                    <li>
+                      {intl.formatMessage({
+                        defaultMessage: "My Community(ies):",
+                        id: "Mw4ShP",
+                        description:
+                          "Label preceding text listing the communities the user is a member of, followed by a colon",
+                      })}{" "}
+                      {nonLegacyIndigenousCommunitySelected
+                        ? indigenousCommunities.map((community, index) => {
+                            if (
+                              !community ||
+                              community ===
+                                IndigenousCommunity.LegacyIsIndigenous
+                            ) {
+                              return "";
+                            }
+
+                            const text = intl.formatMessage(
+                              getIndigenousCommunity(community),
+                            );
+
+                            if (index === 0) {
```

Comment:
Instead of manually putting these together, I recommend Array.join. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
Yeah, that cleaned things up a bit. Changed with [9f64ade](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5494/commits/9f64ade46869ac084785bf5c91b7db5f431b912e).

Reply:
:heart: